python/lsst/ts/externalscripts/base_make_calibrations.py

```diff
@@ -1535,66 +1539,69 @@ async def arun(self, checkpoint=False):
-                    # from the individual images just taken, and if
-                    # do_verify=True, the verification task
-                    # will run the tests using calibrations in its
-                    # input collections as reference.
-                    # Note that there is no certification of combined
-                    # calibrations here, because we are not generating
-                    # them.
-                    # job_id_calib should be None
-                    assert job_id_calib is None, "'job_id_calib' is not 'None'."
-                    response_ocps_verify_pipetask = await self.verify_calib(
-                        im_type, job_id_calib
-                    )
-                    job_id_verify = response_ocps_verify_pipetask["jobId"]
-                    # Check verification statistics
-                    report_check_verify_stats = await self.check_verification_stats(
-                        im_type, job_id_verify, job_id_calib
-                    )
-                    # Inform the user about the results from running
-                    # cp_verify.
-                    # TODO: If verification failed, issue an alarm
-                    # in the watcher: DM-33898
-                    await self.analyze_report_check_verify_stats(
-                        im_type,
-                        report_check_verify_stats,
-                        job_id_verify,
-                        job_id_calib,
-                    )
+                        # If combined calibrations are not being generated
+                        # from the individual images just taken, and if
+                        # do_verify=True, the verification task
+                        # will run the tests using calibrations in its
+                        # input collections as reference.
+                        # Note that there is no certification of combined
+                        # calibrations here, because we are not generating
+                        # them.
+                        # job_id_calib should be None
+                        assert job_id_calib is None, "'job_id_calib' is not 'None'."
+                        response_ocps_verify_pipetask = await self.verify_calib(
+                            im_type, job_id_calib
+                        )
+                        job_id_verify = response_ocps_verify_pipetask["jobId"]
+                        # Check verification statistics
+                        report_check_verify_stats = await self.check_verification_stats(
+                            im_type, job_id_verify, job_id_calib
+                        )
+                        # Inform the user about the results from running
+                        # cp_verify.
+                        # TODO: If verification failed, issue an alarm
+                        # in the watcher: DM-33898
+                        await self.analyze_report_check_verify_stats(
```

Comment:
so if you're ignoring the exception, why await the task? Is it not better to just launch it and move on?

Reply:
The way the OCPS is working now, if the script is doing multiple steps (e.g. processing BIAS and DARKS) and it launch and move on, the following steps are going to fail for sure, because the data it needs to process will probably not be available. So I think right now the best thing is to wait for the results to come out and than move forward.